.github/workflows/file-format-check.yml

```diff
@@ -0,0 +1,13 @@
+name: Check asset file names
+on: pull_request
+
+jobs:
+  Filenamecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - shell: bash
+        run: |
+          find assets | grep "\.[A-Z]" && echo "Upper case characters found in file endings" && exit 1
+          find assets -type f -mtime -14 -printf '%f\n' | grep "[_+ ]" && echo "Underscore, plus or space found in file names" && exit 1
```

Comment:
```suggestion
          find assets -type f -printf '%f\n' | grep "[^a-zA-Z0-9-]" && echo "No special characters, except hyphen, are allowed in file names" && exit 1
```

Reply:
@sushmangupta this regex won't work since it also triggers the . in the file name. Feel free to add more special chars to the existing regex.

![image](https://github.com/shopware/docs/assets/8600299/a9ab56c2-a179-4a97-a694-8b392b6868fd)

Reply:
how about including a dot here '[^a-zA-Z0-9-.]' ?